spec/controllers/submissions_controller_spec.rb

```diff
@@ -69,12 +69,6 @@
       expect(Submission.find(s.id).assignment.tc_id).to eq assignment_tc_id
     end
 
-    it 'returns 404 if the submission is not found' do
```

Comment:
I think this got deleted when I was resolving conflicts in some local dev branches. I'll add it back in.